status-resolver/build.gradle

```diff
@@ -23,11 +24,22 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
+}
+
+ext {
```

Comment:
why this single module has it like that? It would be best to introduce probably `Deps` like we have in modular projects and reuse those dependencies

Reply:
1625e84cb028ce7b65cc6bd34201660fe27780f8